src/client/resources/web/js/message_types.mjs

```diff
@@ -24,9 +24,10 @@
  * @typedef {BaseModServerMessage & {
  *   type: 'chatMessage',
  *   payload: {
- *     history?: boolean,
```

Comment:
It looks like this is always present.

Reply:
Ah yeah, it wasn't initially when I was developing the history functionality.